tests/Tests/Ellipsoid3d.elm

```diff
@@ -0,0 +1,164 @@
+                in
+                Ellipsoid3d.volume ellipsoid
+                    |> Expect.equal (Sphere3d.volume sphere)
+        ]
+
+
+transformations : Test
+transformations =
+    Test.describe "Transforming two objects equally maintains their relative properties"
+        [ Test.fuzz2
+            (Fuzz.tuple ( Fuzz.ellipsoid3d, Fuzz.point3d ))
+            Fuzz.vector3d
+            "contains before/after translation"
+            (\( ellipsoid, point ) vector ->
+                let
+                    beforeTransformation =
+                        Ellipsoid3d.contains point ellipsoid
+
+                    afterTransformation =
+                        Ellipsoid3d.contains
+                            (Point3d.translateBy vector point)
+                            (Ellipsoid3d.translateBy vector ellipsoid)
+                in
+                Expect.equal beforeTransformation afterTransformation
+            )
+        , Test.fuzz2
+            (Fuzz.tuple ( Fuzz.ellipsoid3d, Fuzz.point3d ))
+            Fuzz.plane3d
+            "contains before/after mirroring"
+            (\( ellipsoid, point ) plane ->
+                let
+                    beforeTransformation =
+                        Ellipsoid3d.contains point ellipsoid
+
+                    afterTransformation =
+                        Ellipsoid3d.contains
+                            (Point3d.mirrorAcross plane point)
+                            (Ellipsoid3d.mirrorAcross plane ellipsoid)
+                in
+                Expect.equal beforeTransformation afterTransformation
+            )
+        , Test.fuzz2
+            (Fuzz.tuple ( Fuzz.ellipsoid3d, Fuzz.point3d ))
+            (Fuzz.tuple ( Fuzz.axis3d, Fuzz.angle ))
+            "contains before/after rotating"
+            (\( ellipsoid, point ) ( axis, angle ) ->
+                let
+                    beforeTransformation =
+                        Ellipsoid3d.contains point ellipsoid
+
+                    afterTransformation =
+                        Ellipsoid3d.contains
+                            (Point3d.rotateAround axis angle point)
+                            (Ellipsoid3d.rotateAround axis angle ellipsoid)
+                in
+                Expect.equal beforeTransformation afterTransformation
+            )
+        , Test.fuzz2
+            (Fuzz.tuple ( Fuzz.ellipsoid3d, Fuzz.axis3d ))
+            Fuzz.vector3d
+            "signedDistanceAlong before/after translation"
+            (\( ellipsoid, axis ) vector ->
+                let
+                    beforeTransformation =
+                        Ellipsoid3d.signedDistanceAlong axis ellipsoid
+
+                    afterTransformation =
+                        Ellipsoid3d.signedDistanceAlong
+                            (Axis3d.translateBy vector axis)
+                            (Ellipsoid3d.translateBy vector ellipsoid)
+                in
+                Expect.equal True <|
```

Comment:
I think you should be able to replace this with the slightly simpler
```elm
Interval.width afterTransformation |> Expect.quantity (Interval.width beforeTransformation)
```

Reply:
Or to get fancier and actually check the values of both endpoints, perhaps something like
```elm
afterTransformation
    |> Expect.all
        [ Interval.minValue >> Expect.quantity (Interval.minValue beforeTransformation)
        , Interval.maxValue >> Expect.quantity (Interval.maxValue beforeTransformation)
        ]
```

Reply:
Fixed! :smile: https://github.com/ianmackenzie/elm-geometry/pull/142/commits/226969520387c6b7e2e9c6e96b1004f5de2d3619